source/frontend/src/features/leases/add/ReferenceSubForm.tsx

```diff
@@ -7,11 +7,11 @@ import * as Styled from './styles';
 const ReferenceSubForm: React.FunctionComponent = () => {
   return (
     <Section header="Documentation">
-      <SectionField label="Physical lease/license exists" contentWidth="5">
-        <InlineYesNoSelect field="hasPhysicalLicense" />
+      <SectionField label="Physical lease/licence exists" contentWidth="5">
+        <InlineYesNoSelect field="hasPhysicalLicence" />
```

Comment:
hmm - this field "hasPhysicalLicence" and the one a couple lines below have been renamed here, but I don't see the corresponding change in the form model class. Could you please check? I wouldn't want this to cause bugs when adding leases.

If you feel the need to modify the frontend model then perhaps the backend models need to be adjusted as well. However I'd leave the fields as-is and only change verbiage

Reply:
the field prop was not intended to be changed. rolled back.